### DISCUSSION TITLE
GNNExplainer with HeteroData

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#4612] [Q&A]
Is it possible to use GNNExplainer with heterodata and empty node types? I cant feed the heterodata dict to gnn explainer

**Comment by rusty1s** [comment]
We do not have support for `HeteroData` inside `GNNExplainer` yet, I am sorry. This is definitely on my TODO though.

**Reply by DogitoErgoSum** [reply]
That is good to know! Can you give a time prediction? 😀

**Reply by rusty1s** [reply]
Probably around July or August.

**Reply by DogitoErgoSum** [reply]
Awesome!

**Reply by DogitoErgoSum** [reply]
@rusty1s Any progress on this?

**Reply by rusty1s** [reply]
This will likely picked up for the next release cycle - sorry for the delay!

**Reply by DogitoErgoSum** [reply]
Any date prediction?

**Reply by rusty1s** [reply]
We are planning to get this into PyG 2.2. Will work with @RBendias and @wsad1 to integrate.
